client/components/site-stats-sticky-link/index.jsx

```diff
@@ -36,11 +39,23 @@ var SiteStatsStickyLink = React.createClass( {
 		}
 	},
 
+	preload: function() {
```

Comment:
With this approach, the connector could easily wrap [`once`](https://lodash.com/docs#once) around `preload` before passing it as a prop, as a simple way to replace the more stateful `_preloaded = true` mechanism.